style.css

```diff
@@ -0,0 +1,153 @@
+
+body {
+    margin: 0;
+    height: 100vh;
+    width: 100vw;
+    background-color: black;
+    background-size: 100%;
+    background-repeat: no-repeat;
+}
+/* *************************************************************** */
+                /*  all items in header below */
+/* *************************************************************** */
+header {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 15%;
+    font-size: 2rem;
+    color: red;
+    font-family: 'Russo One', sans-serif;
+}
+
+h2{
+    height: 6vh;
+    font-size: 10vmin;
+}
+
+
+#playerSwap .button{
+    box-sizing: border-box;
+    align-items: flex-end;
+    clear: left;
+    height: 5vh;
+    display: flex;
+    justify-content: center;
+    margin-bottom: 3%;
+}
+
+
+input {
+    background-color: red;
+    color: black;
+    cursor: pointer;
+    font-size: 24px;
+}
+
+input.switch {
+    background-color: black;
+    color: red;
+}
+
+/* *************************************************************** */
+            /*  all items in game board below */
+/* *************************************************************** */
+ .container {
+    /*height: 72%;*/
+    display: flex;
+    justify-content: center;
+}
+
+#gameBoard {
+    height: 75vmin;
```

Comment:
Watch out about using vh/vw/vmin on inner elements.  You want to use them more for outer ones, and then inside use things like %